services/src/main/java/org/keycloak/protocol/oid4vc/issuance/signing/JwtSigningServiceProviderFactory.java

```diff
@@ -0,0 +1,56 @@
+package org.keycloak.protocol.oid4vc.issuance.signing;
+
+import org.keycloak.component.ComponentModel;
+import org.keycloak.component.ComponentValidationException;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.protocol.oid4vc.model.Format;
+import org.keycloak.provider.ConfigurationValidationHelper;
+import org.keycloak.provider.ProviderConfigProperty;
+
+import java.util.List;
+
+/**
+ * @author <a href="https://github.com/wistefan">Stefan Wiedemann</a>
+ */
+public class JwtSigningServiceProviderFactory implements VCSigningServiceProviderFactory {
+
+    public static final Format SUPPORTED_FORMAT = Format.JWT_VC;
+    private static final String HELP_TEXT = "Issues JWT-VCs following the specification of https://identity.foundation/jwt-vc-presentation-profile/.";
+
+    @Override
+    public VerifiableCredentialsSigningService create(KeycloakSession session, ComponentModel model) {
+        var keyId = model.get(SigningProperties.KEY_ID.getKey());
```

Comment:
AFAIK, I did not see the use of "var" in Keycloak codebase.
@mposolda using "var" is OK?

Reply:
Fixed in [8375fbe](https://github.com/keycloak/keycloak/pull/26263/commits/8375fbead903ec5fc16676e5c76d5d811b415df7)